main.go

```diff
@@ -33,6 +34,8 @@ func run(args []string, exit func(int)) {
 	cli.ConfigureRemoveCommand(app)
 	cli.ConfigureLoginCommand(app)
 	cli.ConfigureServerCommand(app)
+	cli.ConfigureAddMfaCommand(app)
+	cli.ConfigureRemoveMfaCommand(app)
 
```

Comment:
would list make sense here?

Reply:
Oh I mean we have "add-mfa-device", "rm-mfa-device" but no way to list the mfa devices

Reply:
Reverted my change. You can only have one mfa device per iam user and you probably know if you've added one or not, but happy to add for completeness. Actually the cli has [`list-mfa-devices`](https://docs.aws.amazon.com/cli/latest/reference/iam/list-mfa-devices.html) so will add, the output will be similar

Reply:
added